src/org/traccar/web/CsvBuilder.java

```diff
@@ -0,0 +1,167 @@
+package org.traccar.web;
+
+import java.beans.Introspector;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.traccar.helper.Log;
+import org.traccar.model.MiscFormatter;
+
+public class CsvBuilder {
+
+    private static final String LINE_ENDING = "\r\n";
+    private static final String SEPARATOR = ";";
+    private static final DateTimeFormatter DATE_FORMAT = ISODateTimeFormat.dateTime();
+
+    private StringBuilder builder = new StringBuilder();
+
+    private void addLineEnding() {
+        builder.append(LINE_ENDING);
+    }
+    private void addSeparator() {
+        builder.append(SEPARATOR);
+    }
+    public void addLine(String value) {
```

Comment:
`addField` 100% used in `addLine(Object object)`
`addLine` for primitives actually never used now, I've made it for universality of `addArray`.
I can move code from that functions to `addLine(Object object)` and `addArray(Collection<?> array)` but I do not see any difference and we will win nothing I guess.